build-tools/packages/version-tools/api-extractor.json

```diff
@@ -1,4 +1,16 @@
 {
   "$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
-  "extends": "@fluidframework/build-common/api-extractor-common-strict.json"
+  "extends": "@fluidframework/build-common/api-extractor-common-report.json",
+  "apiReport": {
+    "enabled": true,
+    "reportFolder": "<projectFolder>/api-report/"
+  },
+  "messages": {
+    "extractorMessageReporting": {
+      "ae-unresolved-link": {
+        "logLevel": "warning",
```

Comment:
Are there currently broken links? Can we not reasonably fix them before merging?

Reply:
Yeah, there are currently some types that are exported by the build-tools project that doesn't have api-extractor enabled. I'm leaving this for later; once we're fully moved to the new tools, those types will be moved from the old build-tools project into the CLI project.